includes/taxonomy-bsf-docs-cat.php

```diff
@@ -24,7 +24,7 @@
 		<div class="bsf-page-header">
 			<?php
 				echo '<h1 class="page-title">' . single_cat_title( '', false ) . '</h1>';
-				the_archive_description( '<div class="taxonomy-description">', '</div>' );
+				echo '<div class="tax-breadcrumb">' . do_shortcode( '[wpseo_breadcrumb]' ) . '</div>';
```

Comment:
What if I don't have `wordpress-seo` installed and I don't want to install it?

Reply:
No issue if you do not have `wordpress-seo` Installed, We have just added support to the `wordpress-seo`  breadcrumb.

Reply:
What I am saying is what will it print on the screen when the shortcode does not exist? This is how it will look like - https://cl.ly/no9Y
You should check if that shortcode does exist before printing it on the screen.

Reply:
Also, add a prefix of `bsf` when adding HTML classes so there is no conflict when any CSS is added for this.